stl/inc/xthreads.h

```diff
@@ -83,10 +83,10 @@ enum class _Thrd_result : int { _Success, _Nomem, _Timedout, _Busy, _Error };
 // threads
 _CRTIMP2_PURE _Thrd_result __cdecl _Thrd_detach(_Thrd_t);
 _CRTIMP2_PURE _Thrd_result __cdecl _Thrd_join(_Thrd_t, int*);
-_CRTIMP2_PURE void __cdecl _Thrd_sleep(const _timespec64*);

```

Comment:
Hi,

I think the removal of this line broke the ABI compatibility of `std::thread::sleep_for`.

For more context, see: https://github.com/actions/runner-images/issues/9398